src/FakeItEasy/Core/FakeManager.cs

```diff
@@ -93,7 +94,7 @@ public virtual IEnumerable<ICompletedFakeObjectCall> RecordedCallsInScope
             get { return FakeScope.Current.GetCallsWithinScope(this); }
         }
 
-        internal List<ICompletedFakeObjectCall> AllRecordedCalls
+        internal IEnumerable<ICompletedFakeObjectCall> AllRecordedCalls
         {
             get { return this.recordedCallsField; }

```

Comment:
@thomaslevesque so, you're saying "leave it as it is, except maybe rename the horribly-named field"?

Also, one note:

> I'm not so sure it's a good idea. ConcurrentQueue<T>.GetEnumerator() already does a ToList()

I was going to say that our shim does. But it turns out the real one does too! I'm forever learning things from you.